lib/travis/build/script/julia.rb

```diff
@@ -40,6 +40,9 @@ def setup
             sh.cmd 'CURL_USER_AGENT="Travis-CI $(curl --version | head -n 1)"'
             case config[:os]
             when 'linux'
+              if config[:julia] == 'nightly' && config[:arch] == 'arm64'
+                sh.failure 'Nightly Julia binaries are not available for AArch64'
```

Comment:
Is this a typo?

Reply:
Not as far as I can tell. Which part seems like a typo?

Reply:
Oh, I see. I was ignorant of the "AArch64". :-)

Reply:
Oh, okay. Yeah, Julia's names for things aren't the same as like the Debian names, for example, so we call 64-bit ARM "AArch64." If you think it would be better or less confusing for users, I can change it to match the declared `arch` in the YAML. I just figured I'd what Julia calls it.